R/new-model.R

```diff
@@ -50,6 +50,7 @@ new_model <- function(
 
   # check if file already exists and decide whether to overwrite if it does
   check_for_existing_model(.path, .overwrite)
+  .path <- sanitize_file_extension(.path)
```

Comment:
As I [mentioned before](https://github.com/metrumresearchgroup/bbr/pull/510#pullrequestreview-1029193785), it seems to me that this `sanitize_file_extension` call must happen _before_ the `check_for_existing_model` in order for that call to be valid.